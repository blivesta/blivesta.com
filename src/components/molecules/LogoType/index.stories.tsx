import React from 'react';
import { storiesOf } from '@storybook/react';
import LogoType from '.';

storiesOf(`molecules|LogoType`, module).add(`default`, () => <LogoType siteTitle="blivesta" />);
