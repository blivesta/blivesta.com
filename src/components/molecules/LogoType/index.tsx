import * as React from 'react';
import styled from 'styled-components';

import Logo from '../../atoms/Logo';

const Flex = styled.div`
  display: flex;
  align-items: center;
`;

const LogoWrapper = styled.div`
  width: 32px;
  height: 32px;
  margin-right: ${props => props.theme.spaces[1]};
`;

const H1 = styled.h1`
  font-family: ${props => props.theme.fonts.avantGarde};
  font-weight: 700;
  font-size: ${props => props.theme.fontSizes[3]};
  margin-bottom: 0;
  line-height: 1;
`;

interface LogoTypeProps {
  siteTitle: string;
}

const LogoType = ({ siteTitle }: LogoTypeProps) => (
  <Flex>
    <LogoWrapper>
      <Logo />
    </LogoWrapper>
    <H1>{siteTitle}</H1>
  </Flex>
);

export default LogoType;
