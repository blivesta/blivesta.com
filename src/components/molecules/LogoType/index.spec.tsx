import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LogoType from '.';

describe('LogoType', () => {
  it('snapshot', () => {
    const tree = shallow(<LogoType siteTitle="blivesta" />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
