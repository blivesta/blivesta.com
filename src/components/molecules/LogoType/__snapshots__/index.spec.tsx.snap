// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoType snapshot 1`] = `
<styled.div>
  <styled.div>
    <Logo />
  </styled.div>
  <styled.h1>
    blivesta
  </styled.h1>
</styled.div>
`;
